doc/advanced-configuration/database-backends-configuration.md

```diff
@@ -28,6 +28,9 @@ mysql -h localhost -u user -p -e 'create database mongooseim'
 mysql -h localhost -u user -p mongooseim < mysql.sql
 ```
 
+You should also configure MySQL database in `[MongooseIM root]/rel/files/ejabberd.cfg` file. 
```

Comment:
I think it's enough to say "`ejabberd.cfg` file" instead of the path. The thing is that the ejabberd.cfg file described in `Advanced-configuration.md` is a template for the final config file. This, I think, should be stated in the `Advanced configuration`.

Same applies to next 2 changes.